### PR TITLE
[Fluent 2 iOS] HUD colors & padding update

### PR DESCRIPTION
**Pull request by laminesm**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

HUD colors & padding are updated to match fluent 2 specs. I have created `topPadding` and `bottomPadding` variables since fluent 2 HUD doesn't use a single value for both. 

### Verification

The changes were tested on the demo app.

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| before_swiftui_light | after_swiftui_light |
| before_swiftui_dark | after_swiftui_dark |
| before_swiftui_label_light | after_swiftui_label_light |
| before_swiftui_label_dark | after_swiftui_label_dark |
| before_activity_light | after_activity_light |
| before_activity_dark | after_activity_dark |
| before_failure_light | after_failure_light |
| before_failure_dark | after_failure_dark |
| before_nolabel_light | after_nolabel_light |
| before_nolabel_dark | after_nolabel_dark |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [x] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1189)

### CODE REVIEW

**Comment by harrieshin** [comment]
with the new grey, the color contrast ratio is ok?

**Review by laminesm — COMMENTED** [review]
I just tested it and the color contrast is not right. I'll reach out to design and get a new token for the indicator.

Screen Shot 2022-08-25 at 10 28 52 AM

**Comment by laminesm** [comment]
> with the new grey, the color contrast ratio is ok?

Design asked to set the indicator color to FG2 and ignore the contrast ratio issue for now as they will be revisiting FG2 and FG3 soon.

**Review by huwilkes — APPROVED** [review]

File: ios/FluentUI.Demo/FluentUI.Demo/Demos.swift [hunk 1]
```diff
@@ -27,14 +27,14 @@ struct Demos {
...
-        DemoDescriptor("TypographyTokens", TypographyTokensDemoController.self)
+        DemoDescriptor("TableViewCell", TableViewCellDemoController.self)
     ]
 
     static let fluent2DesignTokens: [DemoDescriptor] = [
         DemoDescriptor("Global Color Tokens", GlobalColorTokensDemoController.self),
         DemoDescriptor("Alias Color Tokens", AliasColorTokensDemoController.self),
-        DemoDescriptor("Shadow Tokens", ShadowTokensDemoController.self)
+        DemoDescriptor("Shadow Tokens", ShadowTokensDemoController.self),
+        DemoDescriptor("Typography Tokens", TypographyTokensDemoController.self)
```
Comment (huwilkes) [inline ios/FluentUI.Demo/FluentUI.Demo/Demos.swift:37]: There's a bunch of changes in here that shouldn't be. I think if you run `git merge fluent2-colors` it should fix that.

**Pull request merged**